code/src/Footer.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+export const Footer = () => {
+    return (
+        <footer>
+            <p>Created by Henrike Wiemker</p>
+            <p>Technigo Bootcamp</p>
+            <p>November 2020</p>
+            <img src={require("./logo.svg")} alt="The Technigo logo"/>
```

Comment:
I had issues with adding images and now you taught me how to add a local image. Thank you!